crates/matrix-sdk/src/oidc/mod.rs

```diff
@@ -893,24 +891,16 @@ impl Oidc {
         .await
         .map_err(OidcError::from)?;
 
-        {
-            let rt_hash = response.refresh_token.as_ref().map(|t| {
-                let mut hasher = DefaultHasher::new();
-                t.hash(&mut hasher);
-                hasher.finish()
-            });
-            let mut hasher = DefaultHasher::new();
-            response.access_token.hash(&mut hasher);
-            let at_hash = hasher.finish();
-            tracing::trace!(
-                "Token refresh: new refresh_token: {rt_hash:?} / access_token: {at_hash}"
-            );
-        }
+        tracing::trace!(
+            "Token refresh: new refresh_token: {:?} / access_token: {}",
+            response.refresh_token.as_ref().map(hash),
+            hash(&response.access_token)
+        );
 
         self.set_session_tokens(SessionTokens {
             access_token: response.access_token.clone(),
             refresh_token: response.refresh_token.clone().or(Some(refresh_token)),
-            latest_id_token: id_token.or_else(|| latest_id_token.cloned()),
+            latest_id_token: latest_id_token.cloned(),
```

Comment:
`latest_id_token` should be owned in the function parameters, it would avoid this unnecessary clone.

Reply:
Sweet, thanks.